### PR TITLE
SEO-171719-Links-With-Redirects-Typescript

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the links with redirects and changing image names and alt text.

Regards,
Beryl.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: TypeScript/NuGet-Packages.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ You should get the private Syncfusion JavaScript NuGet feed URL to install or up
 
     [http://nuget.syncfusion.com/nuget_javascript/nuget/getsyncfusionpackages/javascript](http://nuget.syncfusion.com/nuget_javascript/nuget/getsyncfusionpackages/javascript) 
 
-    ![NuGet feed URL](NuGet_Packages_Images/img1.png)
+    ![NuGet feed URL in Typescript NuGet Packages](nuget_packages_images/typescript-nuget-packages-feed-url.png)
```
[the remark this answers is not in the document]
Comment (berylouma) [reply TypeScript/NuGet-Packages.md:26]: Corrected

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Approved

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**